docs/docs/integrations/vectorstores/databricks_vector_search.ipynb

```diff
@@ -0,0 +1,383 @@
+     "inputWidgets": {},
+     "nuid": "f11b902d-a772-45e0-bbd9-526218b717cc",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import getpass\n",
+    "import os\n",
+    "\n",
+    "os.environ[\"OPENAI_API_KEY\"] = getpass.getpass(\"OpenAI API Key:\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "59b568f3-8db2-427e-9a4a-1df6fa7a1739",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "Split documents and get embeddings."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "b28e1c7b-eae4-4be8-abbd-8433c7557dc2",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "from langchain.document_loaders import TextLoader\n",
+    "from langchain.embeddings import OpenAIEmbeddings\n",
+    "from langchain.text_splitter import CharacterTextSplitter\n",
+    "\n",
+    "loader = TextLoader(\"../../modules/state_of_the_union.txt\")\n",
+    "documents = loader.load()\n",
+    "text_splitter = CharacterTextSplitter(chunk_size=1000, chunk_overlap=0)\n",
+    "docs = text_splitter.split_documents(documents)\n",
+    "\n",
+    "embeddings = OpenAIEmbeddings()\n",
+    "emb_dim = len(embeddings.embed_query(\"hello\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "e8fcdda1-208a-45c9-816e-ff0d2c8f59d6",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "## Setup Databricks Vector Search client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "9b87fff1-99e5-4d9f-aba3-d21a7ccc498e",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "from databricks.vector_search.client import VectorSearchClient\n",
+    "\n",
+    "vsc = VectorSearchClient()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a Vector Search Endpoint"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "vsc.create_endpoint(name=\"vector_search_demo_endpoint\", endpoint_type=\"STANDARD\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "81090f87-3efd-4c1e-9f58-8d6adba7553d",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "## Create Direct Vector Access Index"
```

Comment:
Add content here describing what a Direct Vector Access Index is. We want to make sure customers know the difference versus what a Delta Sync Index does, especially what customers are responsible for bringing for the Direct Vector Access Index to work

Reply:
done!